### PR TITLE
Feature/add edit mode to url

### PR DESCRIPTION
**Pull request by PanopticaRising**

### CODE REVIEW

File: src/Hooks/useQuerystringHelper.ts [hunk 1]
```diff
@@ -56,4 +68,25 @@ export const useQuerystringHelper = () => {
     return {getQuerystring, updateRoute};
 };
 
+// https://stackoverflow.com/questions/8648892/how-to-convert-url-parameters-to-a-javascript-object
+// Functions like _.fromPairs and Object.fromEntries do not handle array cases, and instead overwrite repeated keys.
+// for..of won't work with Iterators unless we use --downleveliteration or target >es5
+const paramsToObject = (params: URLSearchParams) => {
```
Comment (red-tt-bear) [inline src/Hooks/useQuerystringHelper.ts:74]: I wonder if query string parse would do the trick

Comment (red-tt-bear) [reply src/Hooks/useQuerystringHelper.ts:74]: Seems to
```Javscript
> require('querystring').parse('a=1&a=2')
[Object: null prototype] { a: [ '1', '2' ] }
```

**Review by red-tt-bear — APPROVED** [review]
Looks good but we could simplify

File: src/Hooks/useQuerystringHelper.ts (hunk 1 above)
Comment (PanopticaRising) [reply src/Hooks/useQuerystringHelper.ts:74]: Oh, cool, that works.

**Review by doombot-exe — APPROVED** [review]

**Pull request merged**